<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLayoutScaleBarWidgetBase</class>
 <widget class="QWidget" name="QgsLayoutScaleBarWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>729</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Scalebar Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 2px; font-weight: bold; background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>Scalebar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-287</y>
        <width>566</width>
        <height>1061</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout">
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="groupBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Main Properties</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
          <item row="0" column="1" colspan="2">
           <widget class="QgsLayoutItemComboBox" name="mMapItemComboBox"/>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="mStyleComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="mStyleLabel">
            <property name="text">
             <string>St&amp;yle</string>
            </property>
            <property name="buddy">
             <cstring>mStyleComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="mMapLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>&amp;Map</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mMapItemComboBox</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="mGroupBoxUnits">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Units</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
          <item row="2" column="0">
           <widget class="QLabel" name="mUnitLabelLabel">
            <property name="text">
             <string>&amp;Label for units</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mUnitLabelLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="mMapUnitsPerBarUnitLabel">
            <property name="text">
             <string>Label unit multiplier</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>mMapUnitsPerBarUnitSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsDoubleSpinBox" name="mMapUnitsPerBarUnitSpinBox">
            <property name="toolTip">
             <string>Specifies how many scalebar units per labeled unit. For example, if your scalebar units are set to &quot;meters&quot;, a multiplier of 1000 will result in the scalebar labels in kilometers.</string>
            </property>
            <property name="decimals">
             <number>6</number>
            </property>
            <property name="maximum">
             <double>9999999999999.000000000000000</double>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="mUnitsComboBox">
            <property name="toolTip">
             <string>Specifies the underlying units used for scalebar calculations, e.g., &quot;meters&quot; or &quot;feet&quot;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="mUnitsLabel">
            <property name="text">
             <string>Scalebar units</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="mUnitLabelLineEdit">
            <property name="toolTip">
             <string>Text used for labeling the scalebar units, e.g., &quot;m&quot; or &quot;km&quot;. This should be matched to reflect the multiplier above.</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="mUnitLabelLabel_2">
            <property name="text">
             <string>Number format</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>mUnitLabelLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="mNumberFormatPushButton">
            <property name="text">
             <string>Customize</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="mGroupBoxSegments">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Segments</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,0">
          <item row="5" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="mSegmentLabel">
            <property name="text">
             <string>Segments</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QgsSpinBox" name="mNumberOfSubdivisionsSpinBox">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsPropertyOverrideButton" name="mRightSegmentSubdivisionsDDBtn">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout3">
            <item>
             <widget class="QgsDoubleSpinBox" name="mSegmentSizeSpinBox">
              <property name="toolTip">
               <string>Number of scalebar units per scalebar segment</string>
              </property>
              <property name="suffix">
               <string> units</string>
              </property>
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="maximum">
               <double>9999999999999.000000000000000</double>
              </property>
              <property name="showClearButton" stdset="0">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsPropertyOverrideButton" name="mSegmentSizeDDBtn">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout1">
            <item>
             <widget class="QgsSpinBox" name="mSegmentsLeftSpinBox">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string>left </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsPropertyOverrideButton" name="mSegmentsLeftDDBtn">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QRadioButton" name="mFitWidthRadio">
            <property name="text">
             <string>Fit segment width</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QgsDoubleSpinBox" name="mHeightSpinBox">
              <property name="suffix">
               <string> mm</string>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsPropertyOverrideButton" name="mHeightDDBtn">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout5">
            <item>
             <widget class="QgsDoubleSpinBox" name="mMaxWidthSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string> mm</string>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsPropertyOverrideButton" name="mMaxWidthDDBtn">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout4">
            <item>
             <widget class="QgsDoubleSpinBox" name="mMinWidthSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string> mm</string>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsPropertyOverrideButton" name="mMinWidthDDBtn">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout2">
            <item>
             <widget class="QgsSpinBox" name="mNumberOfSegmentsSpinBox">
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string>right </string>
              </property>
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsPropertyOverrideButton" name="mSegmentsRightDDBtn">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QRadioButton" name="mFixedSizeRadio">
            <property name="text">
             <string>Fi&amp;xed width</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QgsDoubleSpinBox" name="mSubdivisionsHeightSpinBox">
              <property name="suffix">
               <string> mm</string>
              </property>
              <property name="maximum">
               <double>999.990000000000009</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsPropertyOverrideButton" name="mSubdivisionHeightDDBtn">
              <property name="text">
               <string>…</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="mSubdivisionsLabel">
            <property name="text">
             <string>Right segments subdivisions</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="mSubdivisionsHeightLabel">
            <property name="text">
             <string>Subdivisions height</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="groupBox_5">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Display</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0">
          <item row="7" column="0">
           <widget class="QLabel" name="mFillSymbol1Label">
            <property name="text">
             <string>Primary fill</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1" colspan="2">
           <widget class="QgsAlignmentComboBox" name="mAlignmentComboBox"/>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QgsSymbolButton" name="mFillSymbol2Button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Fill Style…</string>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="mFontLabel">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="mAlignmentLabel">
            <property name="text">
             <string>Alignment</string>
            </property>
           </widget>
          </item>
          <item row="14" column="1" colspan="2">
           <widget class="QgsFontButton" name="mFontButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QgsDoubleSpinBox" name="mLabelBarSpaceSpinBox">
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string> mm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="mLabelBarSpaceLabel">
            <property name="text">
             <string>Label margin</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QgsSymbolButton" name="mFillSymbol1Button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Fill Style…</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="mFillSymbol2Label">
            <property name="text">
             <string>Secondary fill</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="mDivisionStyleLabel">
            <property name="text">
             <string>Division style</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="mLineStyleLabel">
            <property name="text">
             <string>Line style</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QLabel" name="mLabelVerticalPlacementLabel">
            <property name="text">
             <string>Vertical label placement</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QLabel" name="mLabelHorizontalPlacementLabel">
            <property name="text">
             <string>Horizontal label placement</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1" colspan="2">
           <widget class="QgsSymbolButton" name="mLineStyleButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Line Style…</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QComboBox" name="mLabelVerticalPlacementComboBox"/>
          </item>
          <item row="6" column="0" colspan="3">
           <widget class="QComboBox" name="mLabelHorizontalPlacementComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="mBoxSizeLabel">
            <property name="text">
             <string>Box margin</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QgsDoubleSpinBox" name="mBoxSizeSpinBox">
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string> mm</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="mSubdivisionStyleLabel">
            <property name="text">
             <string>Subdivision style</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1" colspan="2">
           <widget class="QgsSymbolButton" name="mSubdivisionStyleButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Subdivision Style…</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1" colspan="2">
           <widget class="QgsSymbolButton" name="mDivisionStyleButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Division Style…</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsFontButton</class>
   <extends>QToolButton</extends>
   <header>qgsfontbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsPropertyOverrideButton</class>
   <extends>QToolButton</extends>
   <header>qgspropertyoverridebutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qgsscrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBoxBasic</class>
   <extends>QGroupBox</extends>
   <header location="global">qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsLayoutItemComboBox</class>
   <extends>QComboBox</extends>
   <header>qgslayoutitemcombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsAlignmentComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsalignmentcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsSymbolButton</class>
   <extends>QToolButton</extends>
   <header>qgssymbolbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>groupBox</tabstop>
  <tabstop>mMapItemComboBox</tabstop>
  <tabstop>mStyleComboBox</tabstop>
  <tabstop>mGroupBoxUnits</tabstop>
  <tabstop>mUnitsComboBox</tabstop>
  <tabstop>mMapUnitsPerBarUnitSpinBox</tabstop>
  <tabstop>mUnitLabelLineEdit</tabstop>
  <tabstop>mNumberFormatPushButton</tabstop>
  <tabstop>mGroupBoxSegments</tabstop>
  <tabstop>mSegmentsLeftSpinBox</tabstop>
  <tabstop>mSegmentsLeftDDBtn</tabstop>
  <tabstop>mNumberOfSegmentsSpinBox</tabstop>
  <tabstop>mSegmentsRightDDBtn</tabstop>
  <tabstop>mFixedSizeRadio</tabstop>
  <tabstop>mSegmentSizeSpinBox</tabstop>
  <tabstop>mSegmentSizeDDBtn</tabstop>
  <tabstop>mFitWidthRadio</tabstop>
  <tabstop>mMinWidthSpinBox</tabstop>
  <tabstop>mMinWidthDDBtn</tabstop>
  <tabstop>mMaxWidthSpinBox</tabstop>
  <tabstop>mMaxWidthDDBtn</tabstop>
  <tabstop>mHeightSpinBox</tabstop>
  <tabstop>mHeightDDBtn</tabstop>
  <tabstop>mNumberOfSubdivisionsSpinBox</tabstop>
  <tabstop>mRightSegmentSubdivisionsDDBtn</tabstop>
  <tabstop>mSubdivisionsHeightSpinBox</tabstop>
  <tabstop>mSubdivisionHeightDDBtn</tabstop>
  <tabstop>groupBox_5</tabstop>
  <tabstop>mBoxSizeSpinBox</tabstop>
  <tabstop>mLabelBarSpaceSpinBox</tabstop>
  <tabstop>mLabelVerticalPlacementComboBox</tabstop>
  <tabstop>mLabelHorizontalPlacementComboBox</tabstop>
  <tabstop>mFillSymbol1Button</tabstop>
  <tabstop>mFillSymbol2Button</tabstop>
  <tabstop>mLineStyleButton</tabstop>
  <tabstop>mDivisionStyleButton</tabstop>
  <tabstop>mSubdivisionStyleButton</tabstop>
  <tabstop>mAlignmentComboBox</tabstop>
  <tabstop>mFontButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
